.github/workflows/validate-json-schema.yml

```diff
@@ -6,14 +6,16 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: intelligence-ai/jsonschema@v0.2.0
-      - run: jsonschema fmt --check --verbose
-      - run: jsonschema lint --verbose
+      - uses: intelligence-ai/jsonschema@v0.5.0
+      - run: jsonschema fmt v2.1 v2.2 v2.3 v2.4 v2.5 v2.6 --check --verbose
```

Comment:
I had to manually specify all of the directories, otherwise `fmt` and `lint` would pick up the tests too

Reply:
This will be easy to forget on new releases...

A global `--ignore` flag would be useful in the command. Or we could have a directory `src` and another `tests` but this has a lot more consequences outside this repository. I'll think about it.

Reply:
An `--ignore` flag makes sense. I'll get it done and update this PR

Reply:
I'm prototyping it here: https://github.com/Intelligence-AI/jsonschema/pull/87

Reply:
Done! `--ignore` is making this a lot better now! Please take a look @alombarte !

Reply:
AWESOME!